pontoon/teams/views.py

```diff
@@ -96,8 +96,15 @@ def ajax_permissions(request, locale):
     project_locales = l.project_locale.available()
 
     if request.method == 'POST':
-        locale_form = forms.LocalePermsForm(request.POST, instance=l, prefix='general')
+        locale_form = forms.LocalePermsForm(
+            request.user,
+            request.POST,
+            instance=l,
+            prefix='general',
+            request=request
```

Comment:
Why pass `request.user` and `request.POST` if you also pass the entire `request` object?

Reply:
its best not to pass request objects anywhere - they should always stay in views

Reply:
I feel that passing `request` object could pollute code in form. `request.user` has a clear objective. I feel that `request` as the object could pollute and make it less readable or find what's going on.